modules/administration/pages/backup-restore.adoc

```diff
@@ -44,7 +44,7 @@ A full backup of the {productname} consists of backing up the following componen
 
 [WARNING]
 ====
-The {productname} service is automatically stopped during creation of a full backup.
+The {productname} service is automatically restarted during creation of a full backup, the downtime can be significant.
```

Comment:
I would prefer something like:

```suggestion
The {productname} service is automatically stopped for the duration of creating of a full backup, the downtime can be significant.
After backup is done, service is automatically restarted.
```

It, in my opinion, better explains what is going on and that it is not an instant restart.

Reply:
Alternate suggestion by keichwa:
```suggestion
The {productname} service is automatically stopped for the time it takes to create a full backup.
The downtime can be significant.
After backup is done, service is automatically restarted.
```